tools/ci_build/github/azure-pipelines/templates/py-linux.yml

```diff
@@ -30,7 +34,13 @@ jobs:
   variables:
     # The build machine pool doesn't have dotnet, so it can't run CG.
     skipComponentGovernanceDetection: true
+    ORT_CACHE_DIR: $(Pipeline.Workspace)/ort_ccache
```

Comment:
How about changing it to $(Agent.TempDirectory)/ccache ? It will ensure the ccache dir will be deleted after each pipeline run.

Reply:
Thanks. Updated